packages/augur-sdk/src/state/getter/Markets.ts

```diff
@@ -202,15 +202,17 @@ export interface MarketOrderBookOrder {
   mySize: string;
 }
 
+export interface MarketOrderBookOrderBook {
```

Comment:
Did this interface get renamed. 

The UI is referencing `Getters.Markets.OrderBook`

Reply:
Yep, it is, but the parent class was already called MarketOrderBook because it contains a market and and orderbook object. I'm open to other naming suggestions on this if you have any ideas.

Reply:
you are talking about this?
```
export interface MarketOrderBook {
  marketId: string;
  orderBook: {
    [outcome: number]: {
      spread: string | null;
      bids: OrderBook[];
      asks: OrderBook[];
    };
  };
}
```
just break out OutcomeOrderBook

```
export interface MarketOrderBook {
  marketId: string;
  orderBook: OutcomeOrderBook
}
```

export interface OutcomeOrderBook {
      spread: string | null;
      bids: OrderBook[];
      asks: OrderBook[];
}
```